tests/sles4sap/hana_cluster.pm

```diff
@@ -60,10 +65,36 @@ sub run {
         barrier_wait "HANA_INIT_CONF_$cluster_name";
         barrier_wait "HANA_CREATED_CONF_$cluster_name";
 
-        # Upload the configuration into the cluster
-        my @crm_cmds = ("crm configure load update $cluster_conf",
-            "crm resource refresh msl_SAPHana_${sid}_HDB${instance_id}",
-            "crm resource maintenance msl_SAPHana_${sid}_HDB${instance_id} off");
+        # Check for ANGI setup
+        if (check_var('ANGI', 'true')) {
+            assert_script_run "su - $sapadm -c 'sapcontrol -nr $instance_id -function StopSystem'";
+            # Setting up SAP HANA HA/DR providers
+            my $hadr_template = "angi_susHanaHADR_AIO.template";
+            assert_script_run "curl -f -v " . autoinst_url . "/data/sles4sap/$hadr_template -o /tmp/$hadr_template";
+            assert_script_run "su - $sapadm -c 'SAPHanaSR-manageProvider --sid $sid --add /tmp/$hadr_template'";
+            # Adding SUDO permissions
+            my $sudo_saphanasr = "# SAPHanaSR-ScaleUp entries for writing srHook cluster attribute and SAPHanaSR-hookHelper\n" .
```

Comment:
What is the purpose of this comment ? The file is only used within test so there is nobody that will read this, right?

Reply:
I mean rather why there was a need to put comment into sudeors file. Nobody will read it as far as I can see.